File: backend/app/models/care_result.py
from pydantic import BaseModel
from typing import Optional

class CareResult(BaseModel):
    wash_temperature: Optional[str] = None
    dryer_safe: Optional[bool] = None
    dry_clean_only: Optional[bool] = None
    bleach_safe: Optional[bool] = None
    iron_safe: Optional[bool] = None
    special_instructions: Optional[str] = None
    material: Optional[str] = None
    confidence: Optional[str] = None
    raw_summary: str

File: frontend/src/types/CareResult.ts
export interface CareResult {
    wash_temperature: string | null;
    dryer_safe: boolean | null;
    dry_clean_only: boolean | null;
    bleach_safe: boolean | null;
    iron_safe: boolean | null;
    special_instructions: string | null;
    material: string | null;
    confidence: string | null;
    raw_summary: string;
}